jobs/build-webapp.groovy

```diff
@@ -531,29 +530,8 @@ def deployToService(service) {
    }
 }
 
-
-// This should be called from within a node().
-// HACK: This is a workaround to manage a bug in our current deploy system,
-// where running gradlew in two services in parallel causes a conflict in
-// the cache dir (See INFRA-3594 for full details). Here we're careful to
-// build kotlin services in series. Once this bug is resolved, we can remove
-// this, and let kotlin services default to the shared deployToService again.
-def deployToKotlinServicesAndDataflow() {
-   for (service in SERVICES) {
-      if (service in ['course-editing']) {
-         deployToService(service);
-      }
-      if (service == 'dataflow-batch') {
-         deployToDataflow();
-      }
-   }
-}
-
-// This should be called from within a node().
-// Similar to Kotlin services which are deployed by gradlew, parallelism is not
-// supported -- we need to ensure that the services are deployed sequentially.
-// Hence this is actually invoked by deployToKotlinServicesAndDataflow() until
-// the gradlew issue (INFRA-3594) is resolved.
+// This should be called from within a node(). Cannot be run in parallel with  
+// another gradlew due to a conflict in the cache dir.
 def deployToDataflow() {
```

Comment:
You can get rid of this too -- dataflow-batch is no more!  I removed it from webapp but forgot to remove it from jenkins.